test/sds/SDS_Regression.ipynb

```diff
@@ -283,6 +283,10 @@
     "    else:\n",
     "        variable_slice = masked_variable\n",
     "\n",
+    "    # Some data are lon, lat rather than lat, lon (Python is row, column)\n",
+    "    if variable_slice.shape == (len(x_values[:]), len(y_values[:])):\n",
+    "        variable_slice = variable_slice.T\n",
+    "\n",
```

Comment:
Yeah - that's a really good question. Right now this function is mainly being used for plotting purposes specifically within these tests. So I guess I could make a (weak) argument that I know the regression tests aren't requesting a square region of any arrays.

Ideally, we could/should replace the plotting with some reference images. I'd perhaps argue that's outside the scope of this ticket, too. But open to push-back.

Reply:
Put it in never/later land. I just didn't really get how you know when to transpose the variable.

Reply:
I think DAS-1500 is a good time to implement more sizeable changes - like comparisons to reference images. (That's the "split the big single notebook into a notebook per service" ticket). I think a comparison to a reference image is actually the better and more automated test, so I'll add an AC to do that. (That was IP sprint work for me, but it will likely be _next_ IP, now 😞)